bin/node-template/node/Cargo.toml

```diff
@@ -51,12 +51,13 @@ sc-network = { version = "0.9.0", path = "../../../client/network" }
 sc-sync-state-rpc = { version = "0.9.0", path = "../../../client/sync-state-rpc" }
 sp-keystore = { version = "0.9.0", path = "../../../primitives/keystore" }
 
-dusk-plonk = { path = "../../../client/plonk" }
+# dusk-plonk = { path = "../../../client/plonk" }
+dusk-plonk = { version = "0.8.2", default-features = false }
```

Comment:
In which cases are we using our fork and in which cases are we using the original dusk-plonk implementation? Can we make it uniform across the project?

Reply:
We are using `[patch]` section in the `cargo.toml` of the `workspace` . That allows us to **override the `dusk-plonk` version in any crate** built on this `workspace`. 

More info about [Overriding dependencies](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html) and our use-case [working with an unpublished minor version](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#working-with-an-unpublished-minor-version)